src/EditorFeatures/Text/Extensions.cs

```diff
@@ -37,10 +37,10 @@ internal static TextLine AsTextLine(this ITextSnapshotLine line)
             => line.Snapshot.AsText().Lines[line.LineNumber];
 
         public static SourceText AsText(this ITextSnapshot textSnapshot)
-            => SnapshotSourceText.From(workspace: null, textSnapshot);
+            => SnapshotSourceText.From(textBufferCloneServiceOpt: null, textSnapshot);
```

Comment:
Should this try fetching the service like TextBufferContainer does?

Reply:
Unfortunately `ITextSnapshot` doesn't expose any properties to try and fetch from.